cobalt.ini

```
@@ -0,0 +1,2 @@
+[Application]
+EntryPoint=/app/extra/msedge
```

```suggestion
 [Application]
Name=edge
EntryPoint=/app/extra/msedge
WrapperScript=/app/extra/microsoft-edge-beta
ConfigDir=microsoft-edge-beta
ExposePids=optional

[Zypak]
SandboxFilename=msedge-sandbox
ExposeWidevine=true
```

@refi64 @tinywrkb this should be sufficient right? I tested it locally and it worked.

---

@TheEvilSkeleton it seems to be running correctly after applying the suggested changes, though I do see an error like this on startup.

```
(flextop-init:5): GLib-GIO-WARNING **: 17:37:09.522: g_file_enumerator_iterate() created without standard::name
```

A minor regression is that Cobalt doesn't seem to support comments in the flags file.

I also tried a test video with Widevine encryption from [here](https://shaka-player-demo.appspot.com/), and didn't notice any problem.

---

@tinywrkb how are you testing the comments? They seem to be working over here.

The flextop-init error...that's unrelated to Cobalt, it just seems like flextop-init wasn't actually being run before, which could mess with our app install support. Thus, the warning never appeared. I'm working on fixing it right now (but yeah, basically that shouldn't really be consequential).

---

@refi64 oh!, the problem is not with comments, but with empty lines.

---

@tinywrkb ah okay, I've fixed that. That being said, I'm going to wait a bit to see if any other issues crop up before making another release, mostly since this isn't actually fatal (it should continue to read the flags file and just warn on and then skip invalid lines).

---

@refi64 thanks!